dGame/dComponents/Component.h

```diff
@@ -50,5 +50,5 @@ class Component
 	/**
 	 * The entity that owns this component
 	 */
-	Entity* m_Parent;
```

Comment:
Why is this being changed?  I had thought the agreement was to move this to a ref instead of an objectID